src/components/ChatLog.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+    const messages = props.entries.map(message => {
+        return (
+            <section key={message.id}>
```

Comment:
[nit] You might consider moving this `section` inside of `ChatEntry` since you render it for every chat entry.